buildSrc/src/main/kotlin/io/getstream/chat/android/command/utils/Changelog.kt

```diff
@@ -0,0 +1,4 @@
+package io.getstream.chat.android.command.utils
+
+internal const val UNRELEASED_START = "<!-- UNRELEASED START -->"
+internal const val UNRELEASED_END = "<!-- UNRELEASED END -->"
```

Comment:
Could you move it to `MarkdownParser` class?
They are only used on this class

Reply:
I moved all the logic of of MarkDown in the  `io.getstream.chat.android.command.utils.MarkDown.kt`